### PR TITLE
Update README.md for gcp

### PR DESCRIPTION
**Pull request by binoue**
`necogcp` for `host-vm` stopped using `Google App Engine` in #728 .

However, as the changes were not reflected in documents, this pull request updates the documents.

### CODE REVIEW

File: docs/gcp/README.md [hunk 1]
```diff
@@ -71,6 +54,11 @@ necogcp create-instance
 
 If you want to update your existing image, re-run this command.
 
+### Extend time to delete `host-vm` automatically
+
+`host-vm` created by `necogcp create-instance` has a `at` job which deletes `host-vm` instance at certain time.
```
Comment (tapih) [inline docs/gcp/README.md]: I think the default shutdown time should be clearly stated here.

File: docs/gcp/README.md [hunk 2]
```diff
@@ -9,22 +9,12 @@ Requirements
 Overview
 --------
 
-![workflow](http://www.plantuml.com/plantuml/svg/ZP1DJWGX58JtdABISQ6BngWxJCmOntW23u3mGdVaXpGmqTjBbEb5_vXPlbVKzuewf2odN9nbKkKmeRlZE5AquHjlpw-GCNPdvPxZPIAP2IVtrPEN7vOqHYSDpMyUEDuJOGWfzoU7qLUyLrKlYPIiIhVXPM9rLA3ldt3TfXi8N45f61Llw_m89py9-jV2n3_16cUr_oDwKg4YeLPk9dP-8i3vnS1cteMrT8jrOnbbGz5GmSLIMIaDCs_hFETR975kPgcWfD4Rh41ixdm_sv4iLQJCyACPCZkGryNP7m00)
+![workflow](http://www.plantuml.com/plantuml/svg/ZT0xoiCm40JWNgSOp5_yAIH8IXN1a3Fa0CfwaGrw64ioENvSH2dy26btD9-MRLCsKoxU2KCvJcZE2hU9JMRC_Yavc8VZ3eCtbflwvg9mJum-fYndZo4iIA0bBud9B4cpqnNw2wqXvHN_c_a96dy8JD7I2DgqXJxOHKEfNy5QFiBgTglnsxBaOkb0qOybCrBgFzxUzqhjIRfUPfsWf25OR23HSkYAToy0)
```
Comment (zoetrope) [inline docs/gcp/README.md]: :+1:

File: docs/gcp/necogcp-example.yml [hunk 3]
```diff
@@ -10,6 +10,9 @@ app:
     - my-instance-2
 compute:
   machine-type: n1-standard-32
+  auto-shutdown:
```
Comment (zoetrope) [inline docs/gcp/necogcp-example.yml]: Is this setting described in the document?

Comment (binoue) [reply docs/gcp/necogcp-example.yml]: Thank you. I added a small description of the field

**Pull request merged**